macros/sql/get_column_values.sql

```diff
@@ -10,7 +10,7 @@ Returns:
     A list of distinct values for the specified columns
 #}
 
-{% macro get_column_values(table, column, max_records=none, default=none) -%}
+{% macro get_column_values(table, column, sort_column=none, sort_direction=none, max_records=none, default=none) -%}
```

Comment:
I had a comment here but it disappeared:

Let's make these new args at the end of the list, in case anyone was relying on order of arguments when previously using this macro 😬 
```suggestion
{% macro get_column_values(table, column, max_records=none, default=none, sort_column=none, sort_direction=none) -%}
```